.gitignore

```diff
@@ -1,7 +1,12 @@
-target
+/target
 *.iml
 .idea
 .DS_Store
 *.DS_Store
 */.DS_Store
-**/.DS_Store
\ No newline at end of file
+**/.DS_Store
+/.allure
+.allure
+/allure-results
+vcs.xml
+out/
```

Comment:
Add a new blank line at the end of the file

Reply:
done